cli/args/mod.rs

```diff
@@ -17,9 +17,9 @@ use deno_semver::npm::NpmPackageReqReference;
 use indexmap::IndexMap;
 
 pub use deno_config::BenchConfig;
-pub use deno_config::CompilerOptions;
+
 pub use deno_config::ConfigFile;
-pub use deno_config::EmitConfigOptions;
+
```

Comment:
Could you cleanup these leftover empty lines?

Reply:
Sorry, my mistake.